src/gsi/read_avhrr.f90

```diff
@@ -204,13 +238,11 @@ subroutine read_avhrr(mype,val_avhrr,ithin,rmesh,jsatid,&
   rlndsea(3) = 30._r_kind
   rlndsea(4) = 30._r_kind
 
-                                        ! 205,206,207,208 or 209 for NOAA-14,16,16,17 & 18 respectively
+                                        ! 205,206,207,208 or 209 for ! NOAA-14,16,16,17 & 18 respectively
```

Comment:
Why was a second ! added to this comment?  Is NOAA-14,16,16,17 & 18 respectively supposed to be on a separate line?